aiReachability/aiReachability/NetworkMonitor.swift

```diff
@@ -19,12 +22,24 @@ public class NetworkMonitor {
     /// Property that indicates the last cellular connection state change
     public var cellularState: NetworkState = .unknown
 
+    /// Property that defines the listener
+    public var listener: Listener?
+
     /// Main initializer for `NetworkMonitor` class
     /// Triggers monitor to start
     public init() {
         start()
     }
 
+    /// Initializer for `NetworkMonitor` class
+    /// * @param listener
+    /// * A listener to handle new updates.
+    /// * This parameter is mandatory.
```

Comment:
Is this valid Swift header documentation markup?
```suggestion
    /// - Parameter listener: A listener to handle new updates.
    /// This parameter is mandatory.
```

Reply:
Got the init removed from the object as the listener will be kept as public.